src/sentry/static/sentry/app/components/events/groupingInfo/utils.tsx

```diff
@@ -15,7 +15,7 @@ export function hasNonContributingComponent(component: EventGroupComponent | und
 }
 
 export function shouldInlineComponentValue(component: EventGroupComponent) {
-  return component.values.every(value => !isObject(value));
+  return (component.values as EventGroupComponent[]).every(value => !isObject(value));
```

Comment:
wasn't sure on this, but it didn't like calling every on type `EventGroupComponent[] | string[]`

Reply:
[typescript playground](https://www.typescriptlang.org/play?#code/KYDwDg9gTgLgBDAnmYcCiA3YA7GBxKCAVzAGEIBbSbHeAXjgG8BYAKDg7gwEMAbI4AGcAXOiy4CxMpWq0A2gF04AHziCYUAJbYA5ooDcbAL6HWbAMYRs6uMHH5CJOA0Y9+Q0YqNxugn9kQfP0xaSRJyKitaUzY7UMcwADo4qEQACjjcZwA+W3sASn0gA) of this weird one